plugins/outputs/bigquery/bigquery.go

```diff
@@ -0,0 +1,233 @@
+package bigquery
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const (
+	defaultOffSetKey   = "offset-key.json"
+	timeStampFieldName = "timestamp"
+)
+
+var defaultTimeout = 5 * time.Second
+
+const sampleConfig = `	
+  ## Credentials File
+  credentials_file = "/path/to/service/account/key.json"
+  ## GCP Project
```

Comment:
Could you please avoid abbreviations here as some first-time-users might have difficulties to understand what it is about... :-)